package.json

```diff
@@ -43,13 +48,14 @@
     "eslint-plugin-prettier": "^3.0.1",
     "eslint-plugin-react": "^7.12.4",
     "husky": "^1.3.1",
+    "lint-staged": "^8.1.5",
     "prettier": "^1.16.4",
     "ts-node": "^8.0.3",
     "ts-node-dev": "^1.0.0-pre.32",
     "tslint": "^5.14.0",
     "typescript": "^3.3.4000"
-	},
-	"husky": {
+  },
+  "husky": {
     "hooks": {
       "pre-commit": "npm run lint"
```

Comment:
"precommit": "lint-staged"

Reply:
I suppose all hooks can be set just in `scripts`.

Reply:
Regarding `lint-staged` you right, without it adding `lint-staged` miss all value. My bad, seems I forgot.

And about `hooks` placement, placing them in scripts will be deprecated soon.